encoders/numeric/FastICAEncoder/requirements.txt

```diff
@@ -1 +1 @@
-sklearn
+scikit-learn==0.24
```

Comment:
should be changed to `scikit-learn==0.24.0`

Reply:
why? when doing pip show, it shows 0.24

Reply:
https://github.com/scikit-learn/scikit-learn/releases/tag/0.24.0
Maybe it works as well but my pycharm is going crazy when using 0.24